benchmarking/tpcds/ray_entrypoint.py

```diff
@@ -42,7 +42,13 @@ def run(
     )
     args = parser.parse_args()
 
+    d = Path(".")
```

Comment:
Is this relative to the current script, or where the script is ran from? I think this will do the latter, which might not be what you want.

Reply:
This is the ray entrypoint and is running on the ray-head. It will be relative to the root of the submitted working directory.

Reply:
Since that is where ray runs the "entrypoint" from.

Reply:
You can't assume that, since this script could also be run locally from someone's laptop.

If you really want this behavior, then try: https://stackoverflow.com/questions/10174211/how-to-make-an-always-relative-to-current-module-file-path

I would avoid spurious prints here though. Is this for debugging? Why is it printing every sub directory?

Reply:
The printouts were for debugging when I wasn't seeing some directories that should have existed. (The reason was being they weren't being updated because they were being `.gitignore`-d by ray upload).

Reply:
Resolved; data being read directly from S3 now instead of being needed to exist locally.